cde-pentaho5/test-js/components/DashboardComponent-spec.js

```diff
@@ -60,11 +60,18 @@ define([
      */
```

Comment:
Shouldn't we have a test for the negative scenario. Where we're trying to map a private parameter ?